docs/reference/remap.cue

```diff
@@ -225,4 +225,15 @@ remap: #Remap & {
 				"""
 		},
 	]
+
+	how_it_works: {
+		event_data_model: {
+			title: "Event Data Model"
+			body:  """
+				The `remap` transform works with both log and metric events. The data model for log events matches
```

Comment:
I'd suggest:

You can use the `remap` transform with both log and metric events. Log events in the `remap` transform correspond directly to Vector's log schema, which means that the transform has access to the whole event. With metric events, however, the `remap` transform only has access to the event's tags.